### PR TITLE
[CI] Request docker build stable when tag

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -66,6 +66,23 @@ jobs:
...
+        - wget https://raw.githubusercontent.com/plume-lib/trigger-travis/master/trigger-travis.sh
+      deploy:
+        - provider: script
+          script: bash trigger-travis.sh --pro Drakkar-Software OctoBot-Builder $TRAVIS_TOKEN
+          cleanup: false
+          on:
+            repo: "$GH_REPO"
+            branch: "$DEPLOY_BRANCH"
+            tags: true
+            condition: "$TRAVIS_OS_NAME = 'linux'"
```
Comment (GuillaumeDSM) [inline .travis.yml:84]: 👍

Comment (GuillaumeDSM) [reply .travis.yml:84]: Should we also use this for "Linux - Python 3 nightly - Deploy" ?

Comment (Herklos) [reply .travis.yml:84]: I don't understand the question

Comment (GuillaumeDSM) [reply .travis.yml:84]: My bad, I misunderstood the idea

**Pull request merged**